internal/schemautil/helpers_test.go

```diff
@@ -72,12 +74,12 @@ func TestDetermineMixedOrganizationConstraintIDToStore(t *testing.T) {
 
 			got, err := ConvertOrganizationToAccountID(ctx, tt.input, client)
 			if tt.wantErr {
-				assert.Error(t, err)
+				require.Error(t, err)
 
 				return
 			}
 
-			assert.NoError(t, err)
+			require.NoError(t, err)
```

Comment:
Why require? I mean, those cases are stateless. Let it fail as many times as it can.